osb-service/src/main/java/de/evoila/cf/cpi/bosh/MongoDBDeploymentManager.java

```diff
@@ -45,52 +56,55 @@ protected void replaceParameters(ServiceInstance serviceInstance, Manifest manif
         log.debug("Updating Deployment Manifest, replacing parameters");
 
         if (!isUpdate) {
-            HashMap<String, Object> manifestProperties = (HashMap<String, Object>) manifest.getInstanceGroups().stream().findAny().get().getProperties();
-            HashMap<String, Object> mongodb_exporter = (HashMap<String, Object>) manifestProperties.get("mongodb_exporter");
-            HashMap<String, Object> mongodb = (HashMap<String, Object>) manifestProperties.get("mongodb");
-            HashMap<String, Object> auth = (HashMap<String, Object>) mongodb.get("auth");
-            HashMap<String, Object> replset = (HashMap<String, Object>) auth.get("replica-set");
-            HashMap<String, Object> backupAgent = (HashMap<String, Object>) manifestProperties.get("backup_agent");
-
-            if (replset == null)
-                auth.put("replica-set", new HashMap<>());
-
-            UsernamePasswordCredential usernamePasswordCredential = credentialStore.createUser(serviceInstance, CredentialConstants.ROOT_CREDENTIALS, "admin");
-            serviceInstance.setUsername(usernamePasswordCredential.getUsername());
-
-            usernamePasswordCredential = credentialStore.getUser(serviceInstance, CredentialConstants.ROOT_CREDENTIALS);
-            credentialStore.createUser(serviceInstance, DefaultCredentialConstants.BACKUP_CREDENTIALS,
-                    usernamePasswordCredential.getUsername(), usernamePasswordCredential.getPassword());
-
-            UsernamePasswordCredential backupAgentusernamePasswordCredential = credentialStore.createUser(serviceInstance,
-                    DefaultCredentialConstants.BACKUP_AGENT_CREDENTIALS);
-            backupAgent.put("username", backupAgentusernamePasswordCredential.getUsername());
-            backupAgent.put("password", backupAgentusernamePasswordCredential.getPassword());
-
-            mongodb_exporter.put("user", usernamePasswordCredential.getUsername());
-            mongodb_exporter.put("password", usernamePasswordCredential.getPassword());
-            auth.put("user", usernamePasswordCredential.getUsername());
-            auth.put("password", usernamePasswordCredential.getPassword());
-
-            if (!replset.containsKey("keyfile")) {
-                replset.put("keyfile", new RandomString(1024).nextString());
-            }
 
-            if (!properties.containsKey(REPLICA_SET_NAME)) {
```

Comment:
Removed but not added again? On purpose?

Reply:
Not on purpose will add it back again.